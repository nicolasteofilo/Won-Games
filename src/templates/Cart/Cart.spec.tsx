import { screen, render } from '@testing-library/react'
import { Cart } from '.'

describe('<Cart />', () => {
  it('should render correctly', () => {
    const { container } = render(<Cart />)

    expect(container.firstChild).toBeInTheDocument()
  })
})
