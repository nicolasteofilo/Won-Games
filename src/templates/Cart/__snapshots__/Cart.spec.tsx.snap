// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Cart /> should render correctly 1`] = `
.c0 {
  width: 100%;
  max-width: 130rem;
  margin-left: auto;
  margin-right: auto;
  padding-left: calc(3.2rem / 2);
  padding-right: calc(3.2rem / 2);
}

.c1 {
  color: #FAFAFA;
  font-size: 2.0rem;
  padding-left: 0.8rem;
  border-left: 0.7rem solid #3CD3C1;
  font-size: 2.0rem;
}

.c3 {
  margin: 5.6rem auto 3.2rem;
  height: 0.1rem;
  background: rgba(181,181,181,0.3);
  border: 0;
}

.c2 {
  display: grid;
  grid-template-columns: 1fr;
  grid-gap: 4.0rem;
  margin: 4.0rem 0;
}

@media (min-width:768px) {
  .c1 {
    font-size: 2.8rem;
  }
}

@media (min-width:768px) {
  .c3 {
    margin: calc(5.6rem * 2.5) auto 5.6rem;
  }
}

@media (min-width:768px) {
  .c2 {
    grid-template-columns: 2fr 1fr;
  }
}

<div
  data-testid="mock base"
>
  <div
    class="c0"
  >
    <h2
      class="c1"
      color="white"
    >
      My cart
    </h2>
    <div
      class="c2"
    >
      <div
        data-testid="mock cartList"
      />
      <div
        data-testid="mock paymentOptions"
      />
    </div>
    <hr
      class="c3"
    />
  </div>
  <div
    data-testid="mock showcase"
  />
</div>
`;
